lib/faexport.rb

```diff
@@ -400,6 +400,219 @@ def ensure_login!
       end
     end
 
+    # GET /new_submissions.json
```

Comment:
Comment should list all the possible endpoints (json, xml, rss)

Reply:
True, fixed in f3261755bc9f679a5aeb381356b8b96b2e923243